src/accordion/accordion.js

```diff
@@ -61,12 +64,52 @@ export default class Accordion extends React.Component<AccordionPropsT, Accordio
     typeof onChange === 'function' && onChange(newState);
   }
 
+  handleKeyDown(e: KeyboardEvent) {
+    if (this.props.disabled) {
+      return;
+    }
+
+    const itemRefs = this.itemRefs;
+
+    const HOME = 36;
+    const END = 35;
+    const ARROW_UP = 38;
+    const ARROW_DOWN = 40;
+
+    if (e.keyCode === HOME) {
+      const firstItem = itemRefs[0];
+      firstItem.current && firstItem.current.focus();
+    }
+    if (e.keyCode === END) {
+      const lastItem = itemRefs[itemRefs.length - 1];
+      lastItem.current && lastItem.current.focus();
+    }
+    if (e.keyCode === ARROW_UP) {
+      const activeItemIdx = itemRefs.findIndex((item) => item.current === document.activeElement);
+      if (activeItemIdx > 0) {
+        const prevItem = itemRefs[activeItemIdx - 1];
+        prevItem.current && prevItem.current.focus();
```

Comment:
The page scrolls when using the arrow keys, so we should be preventing the default event behavior for each of the arrow keys.

Reply:
Thanks -- the same must be true for `home` and `end`, so they should also get prevent default yea?

Reply:
🤔 strange. Working in ladle. I'll play around with it some more to see if I can reproduce unexpected behavior.